### PR TITLE
task-59814-user-login-logout

### PR DESCRIPTION
**Pull request by nhunght**

### CODE REVIEW

**Comment by nguyenthaison** [comment]
Oke

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -44,5 +58,35 @@ body {
...
+  border: none;
+  border-bottom: 2px solid #4790E3;
+}
+
+.col-md-6 {
+  width: 60%;
+}
+
+.col-md-offset-3 {
+  margin-left:20%;
```
Comment (tqtrung09) [inline app/assets/stylesheets/static_pages.scss]: have space after :

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+  <div id="logo">
+    <p class="logo_left"><%= link_to t("layout.framgia_crb"), "#" %></p>
+  </div>
+  
   <ul>
-    <li id="logo"><%= link_to t("layout.framgia_crb"), "#" %></li>
-    <li><%= link_to t("layout.sign_up"), "#" %></li>
-    <li><%= link_to t("layout.log_in"), "#" %></li>
+    <% if user_signed_in? %>
+
```
Comment (daodat18plus) [inline app/views/layouts/_header.html.erb]: delete this line

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -5,29 +5,43 @@ body {
...
     float: left;
     border-right: none;
+    width: 20%;
+    .logo_left {
+      margin: 10px 10px;
+      a {
+        text-decoration: none;
+        color: #0726AE;
+        font-weight: bold;
+        font-family: Book Antiqua,Palatino,serif;
```
Comment (tqtrung09) [inline app/assets/stylesheets/static_pages.scss]: space after ,

**Comment by daodat18plus** [comment]
OK

**Comment by tqtrung09** [comment]
Ok

**Comment by phornsopheak** [comment]
ok

**Pull request merged**